pkg/centos7/Dockerfile

```diff
@@ -96,8 +96,29 @@ RUN yum install -y bzip2-devel sqlite-devel \
  && yum clean all \
  && rm -rf /var/cache/yum
 
-# use pyenv
-ARG PYENV_VERSION=3.7.9
+
+# OpenSSL build steps
```

Comment:
Can you please add a little more documentation here specifying the need to build openssl. This will be helpful for open source community also

Reply:
done